yarn.lock

```diff
@@ -13891,6 +14127,13 @@ number-is-nan@^1.0.0:
   resolved "https://registry.yarnpkg.com/number-is-nan/-/number-is-nan-1.0.1.tgz#097b602b53422a522c1afb8790318336941a011d"
   integrity sha1-CXtgK1NCKlIsGvuHkDGDNpQaAR0=
 
+numbro@^2.3.6:
```

Comment:
It's not great that this brings in _yet another_ big number library. There's no easy alternative here?

Reply:
we can switch to something else, i don't mind